src/main/java/ch/njol/skript/effects/EffPathfind.java

```diff
@@ -36,11 +36,11 @@
 
 @Name("Pathfind")
 @Description({"Make an entity pathfind towards a location or another entity. Not all entities can pathfind. " +
-	"If the pathfinding target is another entity, the entities may or may not continuously follow the target."})
+	"If the pathfinding target is another entity, the entities may or may not continuously follow the target. This is a Paper Spigot exclusive."})
 @Examples({
 	"make all creepers pathfind towards player",
 	"make all cows stop pathfinding",
-	"make event-entity pathfind towards player"
+	"make event-entity pathfind towards player at speed 1"
 })
 @Since("INSERT VERSION")
 public class EffPathfind extends Effect {
```

Comment:
Since you're editing, the check at L49 is no longer needed since it's 1.13+

Reply:
It's a Paper Spigot exclusive effect.

Reply:
Then that class needs `@RequiredPlugins` annotation instead of the description note